Change "Thriller" query on homepage

### Problem

The Thriller carousel on the homepage queries specific authors which means the same works always appear, and also non-thrillers appear by those authors. 

#### Evidence / Screenshot

#### Relevant URL(s)


### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:


### Context

- Browser (Chrome, Safari, Firefox, etc):
- OS (Windows, Mac, etc):
- Logged in (Y/N): Y
- Environment (prod, dev, local): prod


### Notes from this Issue's Lead

#### Proposal & constraints
<!-- What is the proposed solution / implementation? Is there a precedent of this approach succeeding elsewhere? -->
Change the query to https://archive.org/search?query=openlibrary_work%3A%28*%29+AND+collection%3A%28inlibrary%29+AND+%28lending___available_to_browse%3Atrue+OR+lending___available_to_borrow%3Atrue%29+AND+%28%22thriller%22%29&sort=-downloads. 
#### Related files
<!-- Files related to this issue; this is super useful for new contributors who might want to help! If you're not sure, leave this blank; a maintainer will add them. -->

#### Stakeholders
<!-- @ tag stakeholders of this bug -->

Thanks @mekarpeles , i forked the repo this time and cloned it locally. I am trying to get the local version of open library running. 
I encountered problems related to the stuff mentioned in troubleshooting 
https://github.com/internetarchive/openlibrary/tree/master/docker#fully-resetting-your-environment
which I tried to resolve by running the commands mentioned in troubleshooting. 
But when I run docker compose up , I run into the following error even after resetting the full environment 
`| 172.18.0.3:51010 - - [02/Mar/2025 00:21:00] "HTTP/1.1 GET /openlibrary.org/log/2025-03-02:0" - 200 OK
2025-03-01 16:21:05 infobase-1      | 172.18.0.3:51018 - - [02/Mar/2025 00:21:05] "HTTP/1.1 GET /openlibrary.org/log/2025-03-02:0" - 200 OK
2025-03-01 16:21:05 db-1            | FATAL:  role "openlibrary" does not exist
2025-03-01 16:21:05 web-1           | 2025-03-02 00:21:05 [infobase] [ERROR] Error in processing request GET /
2025-03-01 16:21:05 web-1           | Traceback (most recent call last):
2025-03-01 16:21:05 web-1           |   File "/openlibrary/infogami/infobase/server.py", line 103, in g
2025-03-01 16:21:05 web-1           |     d = f(self, *a, **kw)
2025-03-01 16:21:05 web-1           |         ^^^^^^^^^^^^^^^^^
2025-03-01 16:21:05 web-1           |   File "/openlibrary/infogami/infobase/server.py", line 452, in delegate
2025-03-01 16:21:05 web-1           |     raise RuntimeError(f"get_site({sitename}) failed")
2025-03-01 16:21:05 web-1           | RuntimeError: get_site(openlibrary.org) failed
2025-03-01 16:21:05 web-1           | 2025-03-02 00:21:05 [infobase.ol] [ERROR] Error
2025-03-01 16:21:05 web-1           | Traceback (most recent call last):
2025-03-01 16:21:05 web-1           |   File "/openlibrary/infogami/infobase/server.py", line 103, in g
2025-03-01 16:21:05 web-1           |     d = f(self, *a, **kw)
2025-03-01 16:21:05 web-1           |         ^^^^^^^^^^^^^^^^^
2025-03-01 16:21:05 web-1           |   File "/openlibrary/infogami/infobase/server.py", line 452, in delegate
2025-03-01 16:21:05 web-1           |     raise RuntimeError(f"get_site({sitename}) failed")
2025-03-01 16:21:05 web-1           | RuntimeError: get_site(openlibrary.org) failed
2025-03-01 16:21:05 web-1           | 2025-03-02 00:21:05 [infobase.ol] [ERROR] Error saved to /var/log/openlibrary/infobase/errorlog/2025-03-02/infobase-002105.897972.html
2025-03-01 16:21:10 infobase-1      | 172.18.0.3:35794 - - [02/Mar/2025 00:21:10] "HTTP/1.1 GET /openlibrary.org/log/2025-03-`

--- a/openlibrary/plugins/openlibrary/home.py
+++ b/openlibrary/plugins/openlibrary/home.py
@@ -18,12 +18,6 @@ from openlibrary.utils import dateutil
 logger = logging.getLogger("openlibrary.home")
 
 CAROUSELS_PRESETS = {
-    'preset:thrillers': (
-        '(creator:"Clancy, Tom" OR creator:"King, Stephen" OR creator:"Clive Cussler" '
-        'OR creator:("Cussler, Clive") OR creator:("Dean Koontz") OR creator:("Koontz, '
-        'Dean") OR creator:("Higgins, Jack")) AND !publisher:"Pleasantville, N.Y. : '
-        'Reader\'s Digest Association" AND languageSorter:"English"'
-    ),
     'preset:comics': (
         '(subject:"comics" OR creator:("Gary Larson") OR creator:("Larson, Gary") '
         'OR creator:("Charles M Schulz") OR creator:("Schulz, Charles M") OR '

--- a/openlibrary/plugins/openlibrary/tests/test_home.py
+++ b/openlibrary/plugins/openlibrary/tests/test_home.py
@@ -127,9 +127,7 @@ class TestHomeTemplates:
             "Books We Love",
             "Recently Returned",
             "Kids",
-            "Thrillers",
             "Romance",
-            "Textbooks",
         ]
         for h in headers:
             assert h in html

--- a/openlibrary/plugins/openlibrary/tests/test_home.py
+++ b/openlibrary/plugins/openlibrary/tests/test_home.py
@@ -127,6 +127,7 @@ class TestHomeTemplates:
             "Books We Love",
             "Recently Returned",
             "Kids",
+            "Thrillers",
             "Romance",
             "Textbooks",
         ]

--- a/openlibrary/plugins/openlibrary/tests/test_home.py
+++ b/openlibrary/plugins/openlibrary/tests/test_home.py
@@ -127,7 +127,6 @@ class TestHomeTemplates:
             "Books We Love",
             "Recently Returned",
             "Kids",
-            "Thrillers",
             "Romance",
             "Textbooks",
         ]

--- a/openlibrary/templates/home/index.html
+++ b/openlibrary/templates/home/index.html
@@ -34,7 +34,7 @@ $add_metatag(name="twitter:card", content="homepage_summary")
 
   $if not test:
     $:macros.QueryCarousel(query="subject:thrillers ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Thrillers'), key="thrillers", url="/subjects/thrillers", sort='random.hourly', use_cache=False)
-    $:macros.QueryCarousel(query="subject:textbooks ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Textbooks'), key="textbooks", url="/subjects/textbooks", sort='random.hourly', use_cache=False)
+    $:macros.QueryCarousel(query="subject_key:textbooks publish_year:[1990 TO *] readinglog_count :[5 TO *] ebook_access:[borrowable TO *]", title=_('Textbooks'), key="textbooks", url="/subjects/textbooks", sort='random.hourly', use_cache=False)
 
   $:render_template("home/custom_ia_carousel", title="Authors Alliance & MIT Press", key="authorsalliance_mit_press", query="preset:authorsalliance_mitpress", limit=18, test=test)
 

--- a/openlibrary/templates/home/index.html
+++ b/openlibrary/templates/home/index.html
@@ -32,7 +32,7 @@ $add_metatag(name="twitter:card", content="homepage_summary")
 
   $:render_template("home/custom_ia_carousel", title=_('Kids'), key="children", query="subject:(Juvenile Fiction)", sorts=["lending___last_browse desc"], limit=18, test=test)
 
-  $:render_template("home/custom_ia_carousel", title=_('Thrillers'), key="thrillers", query="preset:thrillers", sorts=["downloads desc"], limit=18, test=test)
+  $:macros.QueryCarousel(query="subject:thrillers ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Thrillers'), key="thrillers", url="/subjects/thrillers", sort='random.hourly', use_cache=False)
 
   $:render_template("home/custom_ia_carousel", title=_('Textbooks'), key="textbooks", subject="textbooks", sorts=["lending___last_browse desc"], limit=18, test=test)
 

--- a/openlibrary/templates/home/index.html
+++ b/openlibrary/templates/home/index.html
@@ -32,10 +32,10 @@ $add_metatag(name="twitter:card", content="homepage_summary")
 
   $:render_template("home/custom_ia_carousel", title=_('Kids'), key="children", query="subject:(Juvenile Fiction)", sorts=["lending___last_browse desc"], limit=18, test=test)
 
-  $:macros.QueryCarousel(query="subject:thrillers ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Thrillers'), key="thrillers", url="/subjects/thrillers", sort='random.hourly', use_cache=False)
-
-  $:render_template("home/custom_ia_carousel", title=_('Textbooks'), key="textbooks", subject="textbooks", sorts=["lending___last_browse desc"], limit=18, test=test)
-
+  $if not test:
+    $:macros.QueryCarousel(query="subject:thrillers ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Thrillers'), key="thrillers", url="/subjects/thrillers", sort='random.hourly', use_cache=False)
+    $:macros.QueryCarousel(query="subject:textbooks ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Textbooks'), key="textbooks", url="/subjects/textbooks", sort='random.hourly', use_cache=False)
+  
   $:render_template("home/custom_ia_carousel", title="Authors Alliance & MIT Press", key="authorsalliance_mit_press", query="preset:authorsalliance_mitpress", limit=18, test=test)
 
   $:render_template("home/categories", test=test)

--- a/openlibrary/templates/home/index.html
+++ b/openlibrary/templates/home/index.html
@@ -35,7 +35,7 @@ $add_metatag(name="twitter:card", content="homepage_summary")
   $if not test:
     $:macros.QueryCarousel(query="subject:thrillers ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Thrillers'), key="thrillers", url="/subjects/thrillers", sort='random.hourly', use_cache=False)
     $:macros.QueryCarousel(query="subject:textbooks ratings_count:[10 TO *] ebook_access:[borrowable TO *]", title=_('Textbooks'), key="textbooks", url="/subjects/textbooks", sort='random.hourly', use_cache=False)
-  
+
   $:render_template("home/custom_ia_carousel", title="Authors Alliance & MIT Press", key="authorsalliance_mit_press", query="preset:authorsalliance_mitpress", limit=18, test=test)
 
   $:render_template("home/categories", test=test)

